server/sonar-web/src/main/js/apps/users/groups-view.js

```diff
@@ -35,9 +35,9 @@ export default Modal.extend({
         return `${item.name}<br><span class="note">${item.description}</span>`;
       },
       queryParam: 'q',
-      searchUrl: '/api/users/groups?ps=100&login=' + this.model.id,
-      selectUrl: '/api/user_groups/add_user',
-      deselectUrl: '/api/user_groups/remove_user',
+      searchUrl: window.baseUrl + '/api/users/groups?ps=100&login=' + this.model.id,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)